tests/test-installer

```diff
@@ -62,68 +111,98 @@ for arg_mode in client-server engine; do
     # TODO(rfolco): temporary workaround until we make it distro generic
     sudo mkdir -p $(dirname $cfgfile)
     sudo ./crucible-install.sh \
+      --git-repo ${CRUCIBLE_DIR} \
       --${arg_mode}-registry myregistry.io/crucible \
       --${arg_mode}-auth-file /tmp/auth-file.json \
       --name "Nobody" \
-      --email "nobody@somewhere.com"
+      --email "nobody@somewhere.com" \
+      --verbose
     sudo grep "CRUCIBLE_NAME=\"Nobody\"" $idfile || exit 1
     sudo grep "CRUCIBLE_EMAIL=\"nobody@somewhere.com\"" $idfile || exit 1
+    stop_test
 done
 
 for arg_mode in client-server engine; do
+    start_test
     echo "testing arg_mode=${arg_mode}"
     # override existing installation
     touch /tmp/auth-file.json
     cfgfile=$(grep SYSCONFIG= crucible-install.sh | cut -d '=' -f2 | sed 's/"//g')
     # TODO(rfolco): temporary workaround until we make it distro generic
     sudo mkdir -p $(dirname $cfgfile)
     sudo ./crucible-install.sh \
+      --git-repo ${CRUCIBLE_DIR} \
       --${arg_mode}-registry myregistry.io/crucible \
-      --${arg_mode}-auth-file /tmp/auth-file.json
+      --${arg_mode}-auth-file /tmp/auth-file.json \
+      --verbose
     test "$?" = "0" || exit 1
     ls /opt/crucible-moved* || exit 1
+    stop_test
 done
 
 for arg_mode in client-server engine; do
+    start_test
     echo "testing arg_mode=${arg_mode}"
     # override existing installation, no auth-file arg
     cfgfile=$(grep SYSCONFIG= crucible-install.sh | cut -d '=' -f2 | sed 's/"//g')
     # TODO(rfolco): temporary workaround until we make it distro generic
     sudo mkdir -p $(dirname $cfgfile)
     sudo ./crucible-install.sh \
-      --${arg_mode}-registry myregistry.io/crucible
+      --git-repo ${CRUCIBLE_DIR} \
+      --${arg_mode}-registry myregistry.io/crucible \
+      --verbose
     test "$?" = "0" || exit 1
     ls /opt/crucible-moved* || exit 1
+    stop_test
 done
 
+start_test
 # RELEASE: --release and --list-releases
 ## prep step: check tags in remote repo
 tags=$(git ls-remote --tags \
     --sort='version:refname' \
     https://github.com/perftool-incubator/crucible.git \
     | awk -F/ '{print$NF}')
+stop_test
 
+start_test
 ## show releases, test rc and stdout
 stdout=$(sudo ./crucible-install.sh \
+    --git-repo ${CRUCIBLE_DIR} \
     --list-releases)
 test "$?" = "0" || exit 1
```

Comment:
I like the start/stop idea, wondering if we can improve how the expected output is checked at the end of each test...

Reply:
Yeah, it's an improvement but I feel like it could still be better.  For someone like me who isn't as familiar as looking at this job as you probably are it was quite painful at first just to figure out which test case was actually failing so this was an attempt to fix that.